server/controllers/ticket/search.php

```diff
@@ -105,6 +108,9 @@ public function validations() {
 
     public function handler() {
         $this->ignoreDeparmentFilter = (bool)Controller::request('supervisor');
+        $this->ticketTableExists = !empty(RedBean::getAll("SHOW TABLES LIKE 'ticket';"));
+        $this->tagsTableExists = !empty(RedBean::getAll("SHOW TABLES LIKE 'tag_ticket';"));
+        $this->ticketEventTableExists = !empty(RedBean::getAll("SHOW TABLES LIKE 'ticketevent';"));
```

Comment:
Do we need this checks?
https://github.com/opensupports/opensupports/blob/master/server/data/db_schema.sql
This schema script is executed on installation.

PD: I think the problem relies in having older versions that can updgrade correctly, @guillegiu what did we decide in this case?